### DISCUSSION TITLE
Is there a setting to have the sequence number in the source member displayed?

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#1711] [Q&A]
Every now and then a program fails. When I load up the source, the source sequence is not displayed. Normally, this is okay, however, when something goes wrong, the dump or job log shows the statement in error. In today's example, the statement in error is 801 or 8.01. This forces me to go back to SEU to see the statement in error.

TIA, Darryl Freinkel. 

<hr />

⚠️ **REMOVE THIS LINE AND ANY SENSITIVE INFORMATION BELOW!** ⚠️

|Context|Version|
|-|-|
|Code for IBM i version|2.5.0|
|Visual Studio Code version|1.85.0|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
CL (vscode-clle): 1.1.2
Code Coverage for IBM i (code-coverage-ibmi): 0.1.7
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.4.0
Db2 for IBM i (vscode-db2i): 0.6.0
Emmet (emmet): 1.0.0
Error Lens (errorlens): 3.15.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.6
JSON Language Features (json-language-features): 1.0.0
Merge Conflict (merge-conflict): 1.0.0
Node Debug Auto-attach (debug-auto-launch): 1.0.0
TODO Highlight (vscode-todo-highlight): 1.0.5
TypeScript and JavaScript Language Features (typescript-language-features): 1.0.0
notebook (notebook): 0.0.63
```
</details>

<hr />

<details>
<summary>Remote system</summary>

|Setting|Value|
|-|-|
|IBM i OS|V7R4M0|
|Tech Refresh|7|
|CCSID|65535|
|SQL|Disabled
|Source dates|Enabled

### Enabled features

|/QOpenSys/pkgs/bin|/usr/bin|/QSYS.lib/ILEDITOR.lib|/QSYS.LIB|/QIBM/ProdData/IBMiDebugService/bin|
|-|-|-|-|-|
|bash|attr|GENCMDXML.PGM|QZDFMDB2.PGM|startDebugService.sh|
|chsh|iconv|GETNEWLIBL.PGM|||
|ls|setccsid||||
|md5sum|tar||||
|sort|||||
|stat|||||
|tn5250|||||
</details>

<details>
<summary>Shell env</summary>

```bash
BUILDLIB=QGPL
CURLIB=QGPL
HOME=/home/dfreinkel
HOST=172.17.72.21
LIBLS=DPOBJLIB DPPDATA DPSRCLIB LIBHTTP YAJL QTEMP QGPL OPRUTLLIB DFREINKEL
LOGIN=dfreinkel
LOGNAME=dfreinkel
MAIL=/var/spool/mail/dfreinkel
OLDPWD=/home/DFREINKEL
PASE_USRGRP_LIMITED=N
PATH=/QOpenSys/usr/bin:/usr/ccs/bin:/QOpenSys/usr/bin/X11:/usr/sbin:.:/usr/bin
PWD=/home/dfreinkel
SHELL=/QOpenSys/pkgs/bin/bash
SHLVL=1
SSH_CLIENT=10.27.230.65 51021 22
SSH_CONNECTION=10.27.230.65 51021 172.17.72.21 22
TZ=<EST>5<EDT>,M3.2.0,M11.1.0
USER=dfreinkel
USERNAME=dfreinkel
_=/QOpenSys/usr/bin/env
```
</details>

<details>
<summary>Variants</summary>

```json
{
  "american": "#@$",
  "local": "#@$"
}
```
</details>

<details>
<summary>Errors</summary>

```json
[
  {
    "command": "system \"CRTLIB LIB(ILEDITOR) TEXT('Code for i temporary objects. May be cleared.')\"",
    "code": 255,
    "stderr": "CPF2111: Library ILEDITOR already exists.",
    "cwd": "/home/dfreinkel"
  },
  {
    "command": "system \"DLTOBJ OBJ(ILEDITOR/O_*) OBJTYPE(*FILE)\"",
    "code": 255,
    "stderr": "CPF2125: No objects deleted.",
    "cwd": "/home/dfreinkel"
  }
]
```
</details>

**Comment by worksofliam** [comment]
I'm afraid we don't do anything with the source sequence numbers.
